Error on attempt to install wdio wait for with wdio cli

**Environment (please complete the following information):**
 - "@wdio/cli": "^7.5.7"

**Log**
```
✗ npx wdio config

=========================
WDIO Configuration Helper
=========================

? Where is your automation backend located? I have my own Selenium cloud
? What is the IP or URI to your Selenium standalone or grid server? localhost
? What is the port which your Selenium standalone or grid server is running on? 4444
? What is the path to your browser driver or grid server? /wd/hub
? Which framework do you want to use? mocha
? Do you want to use a compiler? TypeScript (https://www.typescriptlang.org/)
? Where are your test specs located? 
? Do you want WebdriverIO to autogenerate some test files? Yes
? Do you want to use page objects (https://martinfowler.com/bliki/PageObject.html)? Yes
? Where are your page objects located? ./test/pageobjects/**/*.ts
? Which reporter do you want to use? spec, allure
? Do you want to add a service to your test setup? wait-for
? What is the base url? http://localhost

Installing wdio packages:
- @wdio/local-runner
- @wdio/mocha-framework
- @wdio/spec-reporter
- @wdio/allure-reporter
- wdio-wait-for-service
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/wdio-wait-for-service - Not found
npm ERR! 404 
npm ERR! 404  'wdio-wait-for-service@*' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/oleksandrkhotemskyi/.npm/_logs/2021-05-14T08_26_27_132Z-debug.log
An unknown error happened! Please retry installing dependencies via "npm i --save-dev @wdio/local-runner @wdio/mocha-framework @wdio/spec-reporter @wdio/allure-reporter wdio-wait-for-service"

Error: unknown
```

--- a/packages/wdio-cli/src/constants.ts
+++ b/packages/wdio-cli/src/constants.ts
@@ -104,6 +104,7 @@ export const SUPPORTED_PACKAGES = {
         { name: 'firefox-profile', value: '@wdio/firefox-profile-service$--$firefox-profile' },
         { name: 'crossbrowsertesting', value: '@wdio/crossbrowsertesting-service$--$crossbrowsertesting' },
         // external
+        { name: 'wait-for', value: 'wdio-wait-for$--$wait-for' },
         { name: 'lambdatest', value: 'wdio-lambdatest-service$--$lambdatest' },
         { name: 'zafira-listener', value: 'wdio-zafira-listener-service$--$zafira-listener' },
         { name: 'reportportal', value: 'wdio-reportportal-service$--$reportportal' },
@@ -121,8 +122,7 @@ export const SUPPORTED_PACKAGES = {
         { name: 'ywinappdriver', value: 'wdio-ywinappdriver-service$--$ywinappdriver' },
         { name: 'performancetotal', value: 'wdio-performancetotal-service$--$performancetotal' },
         { name: 'aws-device-farm', value: 'wdio-aws-device-farm-service$--$aws-device-farm' },
-        { name: 'ocr-native-apps', value: 'wdio-ocr-service$--$ocr-native-apps' },
-        { name: 'wait-for', value: 'wdio-wait-for-service$--$wait-for' }
+        { name: 'ocr-native-apps', value: 'wdio-ocr-service$--$ocr-native-apps' }
     ]
 } as const
 

